packages/react/src/utils/ast-utils.spec.ts

```diff
@@ -393,7 +393,78 @@ const store = createStore(combineReducer({}));
   });
 });
 
-describe('getComponentName', () => {
+describe('findExportDeclarationsForJsx', () => {
+  [
```

Comment:
Minor: Jest supports `describe.each` like:

```
describe.each([
])('findExportDeclarationsForJsx: $testName', ({ testName, src, expectedName }) => {
  it(`should find the component', () => {
    // ...
  });
});
```

Reply:
Nice!!!! TIL 😅  I ended up using `it.each`, too!